run_tts.py

```
@@ -0,0 +1,346 @@
+                    avg_wer = sum(r['metric_value'] for r in provider_model_results) / len(provider_model_results)
+                    print(f"  {provider_name} - {model}: {avg_wer:,.2f}%")
+
+database_path = "tts/Test cases.csv"  # Update this path as needed
+test_cases = load_test_cases(database_path)
+
+if not test_cases:
+    print("No test cases found. Please check your file.")
+else:
+    print(f"Loaded {len(test_cases)} test cases")
+    
+    random_test_case = random.choice(test_cases)
```

Ruff (0.15.7)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)